rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -308776,7 +309045,7 @@
 				"Weight": 15
 			}
 		},
-		"Raptor": {
+		"The Raptor#Fort Forinthry": {
```

Comment:
Raptor does not show up on Slayer tab in the picker currently. Could this be related to the NPC being task-locked behind Unwelcome Guests, or am I just missing something obvious?